newsmatch-donation.php

```diff
@@ -1,117 +1,30 @@
 <?php
-/*
+/**
 Plugin Name: News Match Donation Shortcode
 Plugin URI: http://fairwaytech.com
 Description:  Provides methods to integrate with the donation application hosted at checkout.voiceofsandiego.org
-Version: 1.0
+Version: 0.1
```

Comment:
Why release an `0.1` instead of a `1.0` ?

Reply:
Because right now this is still in development. I'll bump it to 1.0 when we're ready to announce.

Reply:
👍